ncard_app/templates/detail_views/add_award.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'events/base.html' %}
+{% load crispy_forms_tags %}
+{% load static %}
+<link rel="stylesheet" href="{% static 'style.css' %}" type="text/css" media="screen" >
```

Comment:
style.css is already included in the base HTML template. It is redundant to include it here too. Same goes for other form template files where these two lines are included